app/views/questions/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class='clear'><%= image_tag "logo.png",size: "45x85", class: "main_logo"  %>
+<h1 class="title_bar"><span class="logo_bar">fridge</span>overflow</h1></div>
+
+<%= render 'layouts/nav' %>
+
+<table id='question_index'>
+  <thead>
+    <th class='short-column'>Votes</th>
```

Comment:
if you want to use more semantic class names, then you can "abstract" things with mixins (in SASS).